main.js

```diff
@@ -80,6 +80,760 @@ function makeMove(element) {
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="68"></div></td>
+        </tr>
+        <tr>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="71"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="72"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="73"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="74"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="75"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="76"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="77"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="78"></div></td>
+        </tr>
+        <tr>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="81"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="82"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="83"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="84"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="85"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="86"></div></td>
+            <td><div class="BlackBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="87"></div></td>
+            <td><div class="BoardBlock blockDimension" onclick="makeMove(this)"><img onclick="select(this.id)" id="88"></div></td>
+        </tr>
+
+    </table>`;
+
+        document.getElementsByClassName("Board_div")[0].innerHTML = boardHTML;
+        for (let i = 0; i < this.pieces.length; i++) {
+            let piece = this.pieces[i];
+            document.getElementById(piece.getPosition()).src = piece.getSource();
+        }
+
+        // TESTING PURPOSES
+        // var cols = document.getElementsByClassName('ChessBoard')[0].getElementsByTagName('td');
+        // for(let i = 0; i < cols.length; i++){
+        //     // console.log(cols[i].childNodes[0].lastElementChild.id);
+        //     document.getElementsByTagName("td")[i].innerHTML += cols[i].childNodes[0].lastElementChild.id
+        // }
+    }
+}
+
```

Comment:
Didn't we create separate this into different files? Why did you put it back to the main file?